docs/reference/api/hub_registry_spec.md

```diff
@@ -20,4 +20,4 @@ This API is deprecated as of 1.7. To view the old version, see the [go
 here](https://docs.docker.com/v1.7/docker/reference/api/hub_registry_spec/) in
 the 1.7 documentation. If you want an overview of the current features in
 Docker Hub or other image management features see the [image management
-overview](../../userguide/eng-image/image_management.md) in the current documentation set.
+overview](https://docs.docker.com/engine/userguide/eng-image/image_management.md) in the current documentation set.
```

Comment:
should be without `.md`; `https://docs.docker.com/engine/userguide/eng-image/image_management/`